packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -154,8 +154,13 @@ export class Ec2Service extends BaseService implements IEc2Service {
       throw new Error('Maximum percent must be 100 for daemon mode.');
     }
 
-    if (props.daemon && props.minHealthyPercent !== undefined && props.minHealthyPercent !== 0) {
-      throw new Error('Minimum healthy percent must be 0 for daemon mode.');
+    if (props.daemon && props.minHealthyPercent !== undefined && props.maxHealthyPercent !== undefined &&
```

Comment:
This logic doesn't need to be specific to daemons. Your max should always be greater than your min.

Reply:
the problem is if max is equal or smaller than min then it errors out at CFN deployment stage which can have users wait for a long time. But yeah maybe we don't need to check if max > min because this should be a common knowledge. Im just worried if they might set them to be equal to each other then they'd wait until deploying to get the error.

Reply:
you have a check here `if (props.daemon` do you need that condition?